web/models.py

```
@@ -1293,6 +1337,39 @@ class Meta:
+            course = self.session.course
+            if course.max_students:
+                current_enrollments = self.session.enrollments.filter(status='approved').count()
+                
+                # Use the utility function to notify about capacity
+                from .utils import notify_capacity_thresholds
+                notify_capacity_thresholds(
+                    teacher=teacher,
+                    title=self.session.title,
+                    current_count=current_enrollments,
+                    max_count=course.max_students,
+                    item_type="session"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)